src/lib/operators/table_scan/column_is_null_table_scan_impl.cpp

```diff
@@ -50,6 +55,37 @@ void ColumnIsNullTableScanImpl::_scan_generic_segment(const BaseSegment& segment
   });
 }
 
+void ColumnIsNullTableScanImpl::_scan_generic_ordered_segment(const BaseSegment& segment, const ChunkID chunk_id,
```

Comment:
Is this code properly tested?

Reply:
The `table_scan_between_test` permute different `OrderByModes` and `Encodings` (Including e.g. Dictionary, None). Naively I would say yes. But I will check code coverage.

Reply:
The coverage says this code is tested. The `Nulls last` part is not, though. I will add tests accordingly

Reply:
Are you satisfied?